blant.c

```diff
@@ -1907,11 +2047,33 @@ int main(int argc, char *argv[])
 	case 'k': _k = atoi(optarg);
 	    if(!(3 <= _k && _k <= 8)) Fatal("k must be between 3 and 8\n%s", USAGE);
 	    break;
-	case 'w':
-		_windowRep = true;
-		_windowSize = atoi(optarg); 
-		if(_windowSize < _k) Fatal("windowSize must be at least size k\n");
-		break;
+    case 'w':
+        _window = true;
+        _windowSize = atoi(optarg); 
+        if(_windowSize < _k) Fatal("windowSize must be at least size k\n");

```

Comment:
I feel like this should error more gracefully if the window size cannot reasonably fit into memory.

Reply:
Hmm.  If the graph can fit in memory why wouldn't the window?

Reply:
Since I prealloc the 2D array for all combinations (W choose K), it's the 2D table that might not fit into the memory